### PR TITLE
Add a light on the other side

### PR DESCRIPTION
**Pull request by smasahir**
1.A second directional light has been added to the 3D editor. This fix allows you to see the atoms in the model clearly from any direction.
2.Due to various package updates, module import could not be resolved, so Package.json was changed. When changing modules in npm of lts, we imported the recommended version.

### CODE REVIEW

**Comment by timurbazhirov** [comment]
@masahirosaito, @mori0711 Is this ready to be reviewed?

**Comment by mori0711** [comment]
Test @timurbazhirov

**Comment by masahirosaito** [comment]
> @masahirosaito, @mori0711 Is this ready to be reviewed?
Yes, please.

**Comment by masahirosaito** [comment]
@timurbazhirov Should I move the status of the card to "review" in the project board?
I tried to move it, but I can't do so. Perhaps I am not authorized to move the card. Creating card is also.

**Comment by timurbazhirov** [comment]
@masahirosaito - adjusted the permissions, please try again and let me know here

File: src/components/ThreejsEditorModal.js [hunk 1]
```diff
@@ -101,13 +101,25 @@ export class ThreejsEditorModal extends ModalDialog {
...
+    }
+
     /**
      * Load the scene based on the given materials.
      */
     loadScene() {
         const loader = new THREE.ObjectLoader();
         const scene = loader.parse(materialsToThreeDSceneData(this.props.materials));
         this.editor.execute(new window.SetSceneCommand(scene));
+        this.addDirectionalLight();
```
Comment (timurbazhirov) [inline src/components/ThreejsEditorModal.js:122]: The rest of the code looks good, thank you. Here I have a question: where in the code do we add the first light? 

Maybe we should put `addDirectionalLight` there as well?

Comment (masahirosaito) [reply src/components/ThreejsEditorModal.js:122]: The first light, i.e. properties for drawing a shadow caused by the first light, is passed from the default modeler at line 120.
So I think it's a good to put it in this position.

**Pull request closed without merging**